habitat_baselines/common/utils.py

```diff
@@ -5,12 +5,60 @@
 # LICENSE file in the root directory of this source tree.
```

Comment:
This file has become overloaded with a significant amount of code, distributing it even in a simple way across files will help in keeping the modularity. This can be done in this PR or can be added as a TODO for a later PR, @JasonJiazhiZhang  you can take a call on this.

Reply:
Let's do this in a later PR!